src/panels/lovelace/editor/config-elements/hui-entity-filter-card-editor.ts

```diff
@@ -0,0 +1,406 @@
+            return html`
+              <div class="state">
+                <paper-input
+                  .label=${this.hass!.localize(
+                    "ui.panel.lovelace.editor.card.entity_filter.states"
+                  )}
+                  .value=${state as string}
+                  .index=${idx}
+                  @change=${this._stateChanged}
+                >
+                  <mwc-icon-button
+                    .title=${this.hass!.localize(
+                      "ui.panel.lovelace.editor.card.entity_filter.delete_state"
+                    )}
+                    .index=${idx}
+                    slot="suffix"
+                    @click=${this._stateDeleted}
+                  >
+                    <ha-svg-icon .path=${mdiClose}></ha-svg-icon>
+                  </mwc-icon-button>
+                </paper-input>
+              </div>
+            `;
+          })}
+          <paper-input
+            .label=${this.hass!.localize(
+              "ui.panel.lovelace.editor.card.entity_filter.states"
+            )}
+            @change=${this._stateAdded}
+          ></paper-input>
+        </div>
+      </div>
+    `;
+  }
+
+  private _renderCardEditor(): TemplateResult {
+    return html`
+      <div class="card">
+        <ha-formfield
+          .label=${this.hass!.localize(
+            "ui.panel.lovelace.editor.card.entity_filter.show_empty"
+          )}
+          .dir=${computeRTLDirection(this.hass!)}
+        >
+          <ha-switch
+            .checked=${this._config!.show_empty !== false}
+            @change=${this._showEmptyToggle}
+          ></ha-switch>
+        </ha-formfield>
+        ${this._config!.card
+          ? html`
+              <div class="card-options">
+                <mwc-button
+                  @click=${this._toggleCardMode}
+                  .disabled=${!this._cardGUIModeAvailable}
+                  class="gui-mode-button"
+                >
+                  ${this.hass!.localize(
+                    !this._cardEditorEl || this._cardGUImode
+                      ? "ui.panel.lovelace.editor.edit_card.show_code_editor"
+                      : "ui.panel.lovelace.editor.edit_card.show_visual_editor"
+                  )}
+                </mwc-button>
+                <mwc-icon-button
+                  .title=${this.hass!.localize(
+                    "ui.panel.lovelace.editor.edit_card.delete"
+                  )}
+                  @click=${this._deleteCard}
+                >
+                  <ha-svg-icon .path=${mdiDelete}></ha-svg-icon>
+                </mwc-icon-button>
+              </div>
+
+              <hui-card-element-editor
+                .hass=${this.hass}
+                .value=${this._getCardConfig()}
+                .lovelace=${this.lovelace}
+                @config-changed=${this._handleCardConfigChanged}
+                @GUImode-changed=${this._cardGUIModeChanged}
+                .editorOptions=${{ hide_entities: true }}
```

Comment:
Make this a const:
```
const editorOptions = {hide_entities: true};
```
That way the object is not recreated every time it is rendered and will prevent all sub-elements of `hui-card-element-editor` to re-render